examples/integration_test.py

```diff
@@ -353,6 +355,78 @@ def my_on_revoke (consumer, partitions):
     c.close()
 
 
+def verify_stats_cb():
+    """ Verify stats_cb """
+
+    def stats_cb(stats_json_str):
+        stats_json = json.loads(stats_json_str)
+        if 'test' in stats_json['topics']:
+            print("# app_offset stats for topic test partition 0: %d" % stats_json['topics']['test']['partitions']['0']['app_offset'])
+
+    conf = {'bootstrap.servers': bootstrap_servers,
+            'group.id': uuid.uuid1(),
+            'session.timeout.ms': 6000,
+            'error_cb': error_cb,
+            'stats_cb': stats_cb,
+            'statistics.interval.ms': 200,
+            'default.topic.config': {
+                'auto.offset.reset': 'earliest'
+            }}
+
+    c = confluent_kafka.Consumer(**conf)
+    c.subscribe(["test"])
+
+    max_msgcnt = 1000000
+    bytecnt = 0
+    msgcnt = 0
+
+    print('Will now consume %d messages' % max_msgcnt)
+
+    if with_progress:
+        bar = Bar('Consuming', max=max_msgcnt,
+                  suffix='%(index)d/%(max)d [%(eta_td)s]')
+    else:
+        bar = None
+
+    while True:
```

Comment:
Looks good, but this loop doesnt actually verify that stats were received.
I think it could probably finish on the first complete stats seen to speed things up.